### PR TITLE
Final version

### PR DESCRIPTION
**Pull request by jubaan**
Final version for review

### CODE REVIEW

**Comment by Anna-Myzukina** [comment]
### Pull Request under the review by Anna Muzykina (TSE)

**Comment by Anna-Myzukina** [comment]
### Hi team!
### Wow! The project looks really very good! The readme file is so descriptive! :+1:  

### But your project needs changes.
Kindly recommended checking HTML markup with this validator https://validator.w3.org/#validate_by_upload.

screenshot

Now it shows error in your project

ERROR

#### In navbar should be "Hamburger button" & the NY Times image logo embedded in the center 

navbar

* Also this part seems has little problems, kindly recommended fix it.

bottom





### Make changes and submit another code review request.

*If you have any questions, let me know on slack (@Anna-Muzykina) and I will be glad to clarify with you.*

File: main.css
Comment (Anna-Myzukina) [inline main.css]: Don`t repeat your code. Please make changes in main.css file



You can read more here => [Good and Bad CSS Practices for Beginners](https://speckyboy.com/good-bad-css-practices/)

screen

**Review by Anna-Myzukina — CHANGES_REQUESTED** [review]
The project needs changes

File: main.css
Comment (jubaan) [reply main.css]: Hi @Anna-Myzukina, thank you for your great feedback and the link for good and best css practices!! it surely will help us to code better!, I'll send back the corrections asp, i just have a doubt about this last issue "dont repeat your code" the image seems like it has the same code but they are all diferent weight fonts, do i still have to change it? thank you again for your feed back and quick response! :)

**Comment by jubaan** [comment]
Hi Anna, also i wanted to know if you could elaborate with the "hamburger menu" observation i didnt quite understand the issue. thanks again for your great feedback! :)

**Review by nikasulo — CHANGES_REQUESTED** [review]
# Great work implementing some of the previously requested changes :rocket: 
Your project is really beautiful guys, just like the real thing! :smile:. There is however one more thing you definitely need to work on before getting an approval :+1: 

#### HTML Validations
Your markup still has some errors, consider parsing your HTML through [this tool](https://validator.w3.org) to validate and make sure it is semantically correct.
screenshot

##### Please implement this and request a re-review :handshake:

**Review by izaiasneto4 — APPROVED** [review]
### Pull request approved :+1:. Great job doing the requested changes guys. 

You can merge and submit the project completion form!

TSE: Izaias Neto

**Pull request merged**
